src/components/PluginDialog/index.vue

```diff
@@ -242,8 +263,37 @@ export default class extends Vue {
 
     this.rules = rules
 
+    // Generate initial data and merge current data
+    let schemaKeys = {}
+    for (let key in schema.properties) {
+      if (schema.properties[key].default) {
+        schemaKeys[key] = schema.properties[key].default
+        continue
+      }
+
+      switch (schema.properties[key].type) {
+        case 'array':
+          schemaKeys[key] = []
+          this.arrayPropertiesLength[key] = [...new Array(this.pluginData[key] ? this.pluginData[key].length : schema.properties[key].minItems).keys()]
+          break
+
```

Comment:
codes clean.

Reply:
I don't understand. Can you tell me in more detail? 😁

Reply:
Please remove the blank line in `switch` statement :)